### ISSUE TITLE
REINFORCE algorithm

### ISSUE THREAD

**Issue by shifentencent** [titu1994/neural-architecture-search#4]
hi @titu1994, i wonder that why the cross_entropy_loss in contriller(RNN) can be as a loss of variables, in the Williams's paper(simple statistical gradient-following algorithems for connectionist reinforement learning) ,the mass function of the classifier's out sholud be as loss. can you help me check it? thanks~

**Comment by titu1994** [comment]
This was my first tried at reinforcement learning. I'm guessing there must be quite a few odd things. Especially, I used the input as the one hot class label from the last step whereas it should be an embedding. 

This is only trial and very flaky code. For a proper version, see the Tensorflow ENAS project which is open sourced by the authors
